### PR TITLE
Improve import tooling

### PR DESCRIPTION
**Pull request by webmaster128**
With this merged, imports are sorted automatically as part of the eslint autofix step. For me this saves a lot of time that is usually needed to manually sort imports.

### CODE REVIEW

File: packages/iov-cli/src/tsrepl.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { diffLines } from "diff";
 import { join } from "path";
 import { Recoverable, REPLServer, start } from "repl";
-import { register, Register, TSError } from "ts-node";
+import { Register, register, TSError } from "ts-node";
```
Comment (willclarktech) [inline packages/iov-cli/src/tsrepl.ts]: 🤔

File: packages/iov-crypto/src/libsodium.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 // use require instead of import because of this bug
 // https://github.com/jedisct1/libsodium.js/issues/148
```
Comment (willclarktech) [inline packages/iov-crypto/src/libsodium.ts]: Probably time to remove this comment?

File: package.json [hunk 3]
```diff
@@ -34,6 +35,7 @@
     "eslint": "^5.16.0",
     "eslint-config-prettier": "^4.3.0",
     "eslint-plugin-prettier": "^3.1.0",
+    "eslint-plugin-simple-import-sort": "^4.0.0",
```
[the remark this answers is not in the document]
Comment (webmaster128) [reply package.json]: There has never been a good reason for the @iov/* block other than some sense of beauty. I think we can live without that at no cost.

File: packages/iov-cli/src/tsrepl.ts (hunk 1 above)
Comment (webmaster128) [reply packages/iov-cli/src/tsrepl.ts]: That's the JS default sort order:

```
> ["register", "Register"].sort()
[ 'Register', 'register' ]
```

No idea why tslint wanted something different before. Maybe tslint allowed both?

**Comment by webmaster128** [comment]
> How come we can switch away from import x = require('x')? Was there a change in TS?

If I understand correctly, this is what the TS compiler setting `esModuleInterop: true` allows us to do

**Comment by webmaster128** [comment]
Updated

**Pull request merged**